lib/Maui.GoogleMaps/Maui.GoogleMaps.csproj

```diff
@@ -41,6 +41,9 @@
 		<PackageReference Include="Xamarin.GooglePlayServices.Maps" Version="118.2.0.3" />
 	</ItemGroup>
 
+	<ItemGroup Condition="'$(TargetFramework)' != 'net8.0'">
```

Comment:
nope, everything should be done in the Clustering project, i ain't gonna bring that dependency in this project

Reply:
It has nothing to do with clustering, however the functionality needed for it to work is contained in another package, which happens to have clustering as well. Let’s be consistent and leave it in a separate package as well, as most people really don’t need this

Reply:
If this is in the cluster, people will be forced to have something they don't need to use something they do need, man it's GeoJson, it's just another way to populate pins/polygons and whatever is in the file on the map.
I'm going to make the change, but you know the arguments don't make any sense.

Please approve this item here first so that it doesn't generate conflicts.

https://github.com/themronion/Maui.GoogleMaps/pull/82

Reply:
#82 merged! Yeah, too bad I named the package **Onion.Maui.GoogleMaps.Clustering**. Probably would have been more sense to name it after the native ones - **Onion.Maui.GoogleMaps.Utills**. But we are where we are 🫠

Reply:
I have made the corrections, I will do the same in the other PRs, please approve this quickly if you can, because the other PRs are dependent on this change I needed to make in the cluster.

Don't scare me again. ☠️☠️☠️☠️

Reply:
We can use the ClusteredMap as a regular map too, right?

Reply:
No, because there is no enable/disable flag, the idea is to use only cluster, that's why cluster is something separate from everything, they use their own renderers, if I can resolve this item #62  yes, the cluster becomes a common map, but for now, it is not.

Reply:
But, yeah, everything on the other is compatible, but the pins are currently clustered by definition.

Reply:
Ah I see. Ok, I will probably then create a third class after I merge your stuff called ExtendedMap or whatsoever and move the new stuff there, so that people can use regular pins with geojson and heatmaps. Sounds good?

Reply:
> 
> Please approve this item here first so that it doesn't generate conflicts in other's PR's 
> 


@themronion 👀

Reply:
merge #81